sys/contrib/ck/src/ck_ht.c

```diff
@@ -174,8 +174,8 @@ ck_ht_map_create(struct ck_ht *table, CK_HT_TYPE entries)
 	map->step = ck_cc_ffsll(map->capacity);
 	map->mask = map->capacity - 1;
 	map->n_entries = 0;
-	map->entries = (struct ck_ht_entry *)(((uintptr_t)&map[1] + prefix +
-	    CK_MD_CACHELINE - 1) & ~(CK_MD_CACHELINE - 1));
+	map->entries = (struct ck_ht_entry *)roundup2((uintptr_t)&map[1] + prefix,
```

Comment:
If we want to use strict subject bounds this needs to be map +1, but there are probably lots of other issues that are blocking aggressive bounds.

Reply:
Will change to `map + 1`